bin/jobsub_fetchlog

```diff
@@ -65,6 +65,7 @@ def main():
         help='format for downloaded archive: "tar" (default, compressed with gzip) or "zip"',
         default="tar",
     )
+    parser.add_argument("job_id", nargs="?", help="job/submission ID")
```

Comment:
This should be `--jobid`, like https://github.com/marcmengel/jobsub_lite/blob/master/lib/get_parser.py#L89 specifies.  Could this be the same as the line I linked above, just to standardize the arguments across the tools?

Reply:
So... we take *either*` --jobid 1234@schedd `*or* just `1234@schedd`   the former is "jobid", the latter is "job_id", and after we parse the arguments we set one from the other if it is not set.

Reply:
I did miss that - thanks for pointing it out.  In that case, I'm good with this.